index.ts

```diff
@@ -33,8 +37,11 @@ async function getWorkspaceRoot(): Promise<string> {
 	return process.cwd();
 }
 
-async function readCiReport(workspaceRoot: string): Promise<CiReport> {
+async function readCiReport(workspaceRoot: string): Promise<CiReport | undefined> {
 	const ciReportPath = `${workspaceRoot}/.moon/cache/ciReport.json`;
+	if (!(await fileExists(ciReportPath))) {
+		return;
+	}
```

Comment:
This is the key part of the fix, handling cases where the CI report file doesn't exist or can't be read.